### PR TITLE
Update TagBinding.yaml

### PR DESCRIPTION
**Pull request by eahrend**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/11367

This is now supported by the API 
https://cloud.google.com/resource-manager/reference/rest/v3/tagBindings

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
tags: added `tagValueNamespacedName`, field to `TagBinding` resource
```

### CODE REVIEW

File: mmv1/products/tags/TagBinding.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ examples:
       org_id: :ORG_ID
   - !ruby/object:Provider::Terraform::Examples
     name: 'tag_binding_with_namespaced_value'
-    skip_test: true
+    skip_test: false
```
Comment (shuyama1) [inline mmv1/products/tags/TagBinding.yaml:53]:
```suggestion
```
we can remove it then. It should be default to false

Comment (eahrend) [reply mmv1/products/tags/TagBinding.yaml:53]: Appreciate the feedback, but just added this change a couple minutes ago.

Comment (eahrend) [reply mmv1/products/tags/TagBinding.yaml]: Updated to remove it.

**Comment by eahrend** [comment]
@shuyama1 hey, hate to bother you on this, but I don't have access to the test links the bot posted. Is there any action on my side I need to do?

**Comment by shuyama1** [comment]
Looks like unit tests failed ([`terraform-provider-google-beta-build-and-unit-tests`](https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/6577849414/job/17870381123?pr=9294) and [`terraform-provider-google-build-and-unit-tests`](https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/6577849319/job/17870381026?pr=9294))  due to
```
Error: google-beta/services/tags/resource_tags_test.go:404:6: testAccCheckTagsTagBindingDestroyProducer redeclared in this block
Error: 	google-beta/services/tags/resource_tags_tag_binding_generated_test.go:89:6: other declaration of testAccCheckTagsTagBindingDestroyProducer
```

**Comment by eahrend** [comment]
I apologize for the inconvenience yet again, but what should I do to remedy this?

**Review by shuyama1 — CHANGES_REQUESTED** [review]
@eahrend Sorry that I missed your last comment. Okay, looks like we can't run tags tests in parallel and we need to implement them to run in serial according to the [comment](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/tags/resource_tags_test.go#L17). You'll likely need to implement the test as one of the sub tests of [TestAccTags](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/tags/resource_tags_test.go#L20). This needs to be a handwritten test instead of a generated one.  [testAccTagsTagBinding_tagBindingBasicExample](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/tags/resource_tags_test.go#L375) should be a good example that you can follow

**Pull request closed without merging**